core/src/store/redux/actions.ts

```diff
@@ -0,0 +1,21 @@
+import { ReportActionOptions, ReportErrorActionCreatorWithType, DismissErrorActionCreatorWithType } from "./types";
+import typedActionCreatorFactory from "../../typed-action-creator/factory";
+
+const randomIdentifier = () => Math.random().toString(36).substring(2, 15)
```

Comment:
Out of interest why do we not use `uuid` here?

Reply:
not sure - one less dependency? 🤷‍♂
one for @sroze I would say..

Reply:
I don't remember tbh.